<launch>
    <node pkg="mars-quadcopter" type="turtle_tf_broadcaster_1_avg.py" name="turtle_tf_broadcaster_1" output="screen"></node>
    <node pkg="mars-quadcopter" type="turtle_tf_broadcaster_2_avg.py" name="turtle_tf_broadcaster_2" output="screen"></node>
    <node pkg="mars-quadcopter" type="turtle_tf_broadcaster_3_avg.py" name="turtle_tf_broadcaster_3" output="screen"></node>
</launch>